blog/_posts/DelSquared_RGSoC_blogpost.md

```diff
@@ -0,0 +1,47 @@
+﻿
+---
+ title: DelSquared: Rails Girls Summer of Code 2018 and Us
+ layout: post
+ created_at: Fri Jul 10 2018
+ permalink: blog/DelSquared_RGSoC_blogpost
+ current: blog
+ author: Arushi
+ twitter: arushi1803
+ categories:
+ - student-posts
+ ---
+ **DelSquared: Rails Girls Summer of Code 2018 and Us**

```

Comment:
Please user markdown style headers and add blank lines in between each header and text. The comment applies to all titles and text in the whole file.